src/system/commitUtils.ts

```diff
@@ -0,0 +1,23 @@
+/** not sure about file location. I thought about git/formatters, or git/utils */
+
+/**
+ * use `\n` symbol is presented to split commit message to description and title
+ */
+export const splitGitCommitMessage = (commitMessage?: string) => {
```

Comment:
Let's export this as a normal function rather than a lambda const

Reply:
Ok. Could you explain why? Does it really make sence?